src/device.c

```diff
@@ -139,14 +139,21 @@ device_query_permission_sync (const char *app_id,
         {
           g_variant_builder_add (&opt_builder, "{sv}", "icon", g_variant_new_string ("camera-web-symbolic"));
 
-          title = g_strdup (_("Turn On Camera?"));
-          body = g_strdup (_("Access to your camera can be changed "
-                             "at any time from the privacy settings."));
-
           if (info == NULL)
-            subtitle = g_strdup (_("An application wants to use your camera."));
+            {
+              title = g_strdup (_("Allow to Use the Camera?"));
```

Comment:
```suggestion
              title = g_strdup (_("Allow an Application to Use the Camera?"));
```

Reply:
I think here we are not granting the permission to just one application.